data-prepper-expression/src/main/java/org/opensearch/dataprepper/expression/ParseTreeCoercionService.java

```diff
@@ -42,6 +42,8 @@ public Object coercePrimaryTerminalNode(final TerminalNode node, final Event eve
                 return Float.valueOf(nodeStringValue);
             case DataPrepperExpressionParser.Boolean:
                 return Boolean.valueOf(nodeStringValue);
+            case DataPrepperExpressionParser.Null:
+                return null;
```

Comment:
I like that this works out of box with the operators. But, should this be some sort of special class - like `NullValue`? I suppose this could come later if that provides specific value.

Reply:
Frankly, I have no idea. I will follow what you suggest here.

Reply:
It might not matter. And we can change it later since these are internal classes.